### PR TITLE
nautilus: tests: mgr: tox failures when running make check

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/39530

### CODE REVIEW

File: src/pybind/mgr/orchestrator_cli/run-tox.sh [hunk 1]
```diff
@@ -23,10 +30,17 @@ then
...
 if [ "$WITH_PYTHON2" = "ON" ]; then
-  ENV_LIST+="py27"
+  ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST+=",py3"
+if [ "$WITH_PYTHON3" = "3" ]; then
+  ENV_LIST+="py3,"
 fi
+ENV_LIST=$(echo "$ENV_LIST" | sed -e 's/,$//')
```
Comment (kshtsk) [inline src/pybind/mgr/orchestrator_cli/run-tox.sh]: This line is not required anymore

Comment (smithfarm) [reply src/pybind/mgr/orchestrator_cli/run-tox.sh]: Damn, overlooked that one.

This can be fixed in the next round of changes in master - i.e. when we combine all of these near-duplicate scripts into a single script.

**Comment by smithfarm** [comment]
Unstaged pending merge of https://github.com/ceph/ceph/pull/27947 and cherry-pick of that follow-up fix into this PR.

File: src/pybind/mgr/ansible/run-tox.sh [hunk 2]
```diff
@@ -20,10 +27,15 @@ export CEPH_BUILD_DIR=$CEPH_BUILD_DIR
 source ${MGR_ANSIBLE_VIRTUALENV}/bin/activate
 
 if [ "$WITH_PYTHON2" = "ON" ]; then
-  ENV_LIST+="py27"
+  ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST+="py3"
+if [ "$WITH_PYTHON3" = "3" ]; then
```
Comment (jmolmo) [inline src/pybind/mgr/ansible/run-tox.sh]: Probably there is a good reason to change "WITH PYTHONX" variable domain from "ON/OFF" to "ON/OFF/3" but for me is not clear. Can you explain why this change improves the script? thx in advance

Comment (smithfarm) [reply src/pybind/mgr/ansible/run-tox.sh]: @jmolmo Thanks for your comment! I just opened https://github.com/ceph/ceph/pull/28002 to address it. As to why I embarked on this project in the first place, see https://tracker.ceph.com/issues/39579

File: src/pybind/mgr/orchestrator_cli/run-tox.sh (hunk 1 above)
Comment (smithfarm) [reply src/pybind/mgr/orchestrator_cli/run-tox.sh]: done in #28002

**Comment by smithfarm** [comment]
Leaving unstaged pending review/merge of #28002

**Comment by yuriw** [comment]
[wip-yuri5-testing-2019-05-29-1613-nautilus](https://trello.com/c/q7wFCEp3/622-wip-yuri5-testing-2019-05-29-1613-nautilus)
